src/devices/Amg88xx/ResetType.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Iot.Device.Amg88xx
+{
+    /// <summary>
+    /// Defines the reset types of the reset register (addr: 0x01)
+    /// </summary>
+    public enum ResetType : byte
```

Comment:
same comment as for all enums, does it need to be publically exposed?

Reply:
@Ellerbach The only reason for exposing the enums is that I use them in the unit tests and the unit tests have a different namespace. Any advice?

Reply:
for testing you shouldn't have need to use it at all but if you strongly feel you should or makes it much easier then you can:
- InternalsVisibleToAttribute
- include source files directly in the tests

you should never design APIs based on the tests and you should only think about what is useful for users

Reply:
@krwq I strongly agree to 
> never design APIs based on what you need for testing

I can't agree to 
> for testing you shouldn't have need to use it 

I don't want to duplicate enums for testing or work with any kind of "magic" number in my tests. An enum defining registers or bits is a static entity and usually won't change at any time in future,  As they're rather simple, too, I just do a code review and use them in both application and test. At least this is my experience.  
So, probably the InternalsVisibleToAttribute approach would be a good idea.

Reply:
> I don't want to duplicate enums for testing or work with any kind of "magic" number in my tests

I guess it's not about replicating, it's about direct file reference rather than project reference. So you get access to the internal elements with a compatible namespace.

> InternalsVisibleToAttribute approach would be a good idea.

One or the other approach should work. Looking forward to see which one you choose :-)

Reply:
Even though we usually don’t encourage the use of InternalsVisibleTo, in this case it is totally fine specially because we never end up shipping the individual binding assembly but instead ship the all up iot.device.bindings package which won’t have the attribute